modules/myclasses/server/routes/myclasses.server.routes.js

```diff
@@ -17,6 +20,14 @@ module.exports = function(app) {
     .put(myclasses.update)
     .delete(myclasses.delete);
 
+  // Routing for requests with the user's classes
+  app.route('/api/userclasses').put(userclasses.updateclasses);
+  app.route('/api/userclasses').get(userclasses.getclasses);
+  app.route('/api/userclasses/deleteclass').put(userclasses.deleteclass);
```

Comment:
$http.delete is supposed to be used for delete.

Reply:
In the database I am using findOneAndUpdate() with $pull to remove one of the classes from the user's classes array. I am still performing an update, but just removing an entry in the array.

Reply:
right, I forgot that we can't simply delete the whole data from the db